src/pages/settings/AboutPage/AboutPage.js

```diff
@@ -80,73 +80,69 @@ function AboutPage(props) {
-                            {_.map(menuItems, (item) => (
-                                <MenuItem
-                                    key={item.translationKey}
-                                    title={props.translate(item.translationKey)}
-                                    icon={item.icon}
-                                    iconRight={item.iconRight}
-                                    onPress={() => item.action()}
-                                    shouldBlockSelection={Boolean(item.link)}
-                                    onSecondaryInteraction={
-                                        !_.isEmpty(item.link) ? (e) => ReportActionContextMenu.showContextMenu(CONTEXT_MENU_TYPES.LINK, e, item.link, popoverAnchor) : undefined
-                                    }
-                                    ref={(el) => (popoverAnchor = el)}
-                                    shouldShowRightIcon
-                                />
-                            ))}
-                        </View>
-                        <View style={[styles.sidebarFooter]}>
-                            <Text
-                                style={[styles.chatItemMessageHeaderTimestamp]}
-                                numberOfLines={1}
-                            >
-                                {props.translate('initialSettingsPage.readTheTermsAndPrivacy.phrase1')}{' '}
-                                <TextLink
-                                    style={[styles.textMicroSupporting, styles.link]}
-                                    href={CONST.TERMS_URL}
-                                >
-                                    {props.translate('initialSettingsPage.readTheTermsAndPrivacy.phrase2')}
-                                </TextLink>{' '}
-                                {props.translate('initialSettingsPage.readTheTermsAndPrivacy.phrase3')}{' '}
-                                <TextLink
-                                    style={[styles.textMicroSupporting, styles.link]}
-                                    href={CONST.PRIVACY_URL}
-                                >
-                                    {props.translate('initialSettingsPage.readTheTermsAndPrivacy.phrase4')}
-                                </TextLink>
-                                .
-                            </Text>
-                        </View>
-                    </ScrollView>
-                </>
-            )}
-        </ScreenWrapper>
+        <IllustratedHeaderPageLayout
+            title={props.translate('initialSettingsPage.about')}
+            onBackButtonPress={() => Navigation.goBack(ROUTES.SETTINGS)}
+            illustration={LottieAnimations.Coin}
+            shouldShowCloseButton
+            backgroundColor={themeColors.PAGE_BACKGROUND_COLORS[ROUTES.SETTINGS_ABOUT]}
+            overlayContent={overlayContent}
+        >
+            <ScrollView contentContainerStyle={[styles.flexGrow1, styles.flexColumn, styles.justifyContentBetween]}>
```

Comment:
I don't think we need ScrollView here. IllustratedHeaderPageLayout already includes a ScrollView.